test/edge_shape.h

```diff
@@ -0,0 +1,20 @@
+#ifndef VALHALLA_EDGE_SHAPE_H
+#define VALHALLA_EDGE_SHAPE_H
+
+#include "baldr/graphreader.h"
+#include "baldr/graphtile.h"
+#include "midgard/encoded.h"
+
+using namespace valhalla;
+
+std::string get_shape(baldr::GraphReader& graphreader, const valhalla::baldr::GraphId& edgeid) {
```

Comment:
we should instead add this method to `GraphReader` as a convenience function. no need to orphan a single function in a new file